contracts/AuctionBidder.sol

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity <0.9.0;
+
+import "./Auction.sol";
+import "./CoveragePool.sol";
+import "@openzeppelin/contracts/math/SafeMath.sol";
+
+/// @title AuctionBidder
+/// @notice A contract for auction bidders for buying coverage pool auctions. This
+///         contract offers additional features for bidders to decide if their
+///         requirements for making a purchase are satisfied.
+contract AuctionBidder {
+    using SafeMath for uint256;
+
+    CoveragePool public coveragePool;
+
+    constructor(CoveragePool _coveragePool) {
+        coveragePool = _coveragePool;
+    }
+
+    /// @notice Takes an offer from an auction buyer with a minimum required tokens
+    ///         to seize from the coverage pool.
```

Comment:
We could explain why is it a good idea to use this function (another offer for the auction, claim, or withdrawal from the pool).

Reply:
2565a7a